app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserDevicesScreen.kt

```diff
@@ -74,7 +75,8 @@ private fun OtherUserEmptyDevicesContent() {
         verticalAlignment = Alignment.Top
     ) {
         Text(
-            text = stringResource(id = R.string.label_client_key_fingerprint_not_available).lowercase().capitalize(),
+            text = stringResource(id = R.string.label_client_key_fingerprint_not_available).lowercase()
+                .replaceFirstChar { if (it.isLowerCase()) it.titlecase(Locale.getDefault()) else it.toString() },
```

Comment:
You can do the same by using 

```suggestion
            text = stringResource(id = R.string.label_client_key_fingerprint_not_available).capitalizeFirstLetter(),
```